src/IOHprofiler_problem.hpp

```diff
@@ -126,7 +142,8 @@ template <class InputType> class IOHprofiler_problem {
  
 
 
-  bool optimalFound = false;
+  bool rawOptimalFound = false;
```

Comment:
This is for cases that f(x) is the maximal value of raw function but transformed_f(x) is not the maximal transformed fitness. But this does not happen to  af(x)+b transformation.

Reply:
Isn't this the case for all useful transformations tough? If you have a transformation without this property, it would need to be either a random noise transformation or a function which assigns the same value to different points, which I don't think is a useful transformation (then instance 0 has a different number of global optima than the other instances). 
Either way, if we have two different variables for this, we make it more difficult for the user to figure out which one they need, so if it is not needed for any current or planned transformations, it might be better to just have one variable for now.

Reply:
I completely agree... If the transformation creates new optima as artifacts, such a transformation should be invalid. I think I also mention it somewhere in our JSS paper.